src/Jobs/GenerateCSVJob.php

```diff
@@ -136,6 +136,7 @@ protected function getOutputPath()
         $base = ASSETS_PATH . '/.exports';
         if (!is_dir($base)) {
             mkdir($base, 0770, true);
+            chmod($base, 0770);
```

Comment:
I'm not sure why this would do anything that `mkdir()` hasn't already done

Reply:
`chmod` is absolute (regardless of `umask`), where `mkdir` will take the current `umask` into account.

Reply:
Just had a quick skim read over the manual. It sounds like you shouldn't be using masks in a multithreaded environment. If you want to, we could provide extension points for you do add them. I now think this change is correct, since it will undo any mask modifications to `0770` that were applied in the previous line

Reply:
`mkdir` is affected by the executing user's umask, where `chmod` isn't.

Reply:
Woops, missed the responses before responding.